### DISCUSSION TITLE
Change recyclerView to an editText

### DISCUSSION THREAD

**Discussion by ErickDiazRuiz** [Network-Revolution/DotLocalFinder#1] [Ideas]
Hi Network Revolution, first of all thanks for sharing your project, I was testing it and I realized that the result "ip address" appears in a recyclerView, however I think it would be better if the result would be unique and shown in a textView, because each Device has a unique local name and a unique IP for a local network, in any case I consider that it would beconfusing to see more than one IP for the same local name.

**Comment by NetworkRevolution** [comment]
Thanks for using my app.

There is not a single ipv6 address that mDNS will answer. This is because there is only one hostname for a machine, but the standard allows for multiple ipv6 addresses.

I was also wondering how to display the results. I'm using recyclerview to display the results, but I'm not sure if I should be stunned by a few lines of text.
Isn't it cumbersome to select every single line to copy the results to the clipboard?

How do you use this application?
I use it to find non-temporary global IPv6 addresses. I use it to find a non-temporary global IPv6 address because with this address, I can access this IP from outside my home network. For this reason, I think it is enough if I can easily copy an address that does not start with fe80 and does not contain ff:fe in the interface ID part among the multiple IPv6 answers.

Certainly, there are possible applications where you only want to see the IPv6 global routing prefix or check the IPv4 address, but what are those cases?
